rai/qt/qt.cpp

```diff
@@ -908,7 +997,7 @@ active_status (*this)
 	client_layout->setSpacing (0);
 	client_layout->setContentsMargins (0, 0, 0, 0);
 	client_window->setLayout (client_layout);
-	client_window->resize (320, 480);
+	client_window->resize (500, 620);
```

Comment:
Will this affect the size of the whole wallet?

Reply:
Yeah, and I think that's a good thing, unless there's a good reason to keep it tiny.